tox.ini

```diff
@@ -14,6 +15,8 @@ DJANGO =
     2.0: django20
     2.1: django21
     master: djangomaster
+OPTIONALS =
+    1: optionals
```

Comment:
Can we drop this and just keep the tox.ini as-is?

Reply:
It is required for running tests with (and without) optionals on Travis.
tox-travis uses this then.

Reply:
@blueyed see my comment on how to override without requiring the additional factor.